app/views/home_page.eco

```diff
@@ -20,12 +20,12 @@
   <% else: %>
     <hr />
 
-    <% for {season, total, complete}, i in seasons: %>
+    <% for {season, name, total, complete}, i in seasons: %>
       <!-- TODO: Translate these! -->
 
       <div class="progress">
         <p>
-          <span class="season"><strong>Season <%= season %></strong></span>
+          <span class="season"><strong><%= name %></strong></span>
```

Comment:
this is a merge of already committed code we did for the lost season (like this one https://github.com/zooniverse/Serengeti/commit/722e0bee93d984b3dc215c2ed78be84bb706c330 )
Not sure why it's showing as a diff.